controller/backing_image_controller.go

```diff
@@ -145,13 +145,13 @@ func (bic *BackingImageController) handleErr(err error, key interface{}) {
 	}
 
 	if bic.queue.NumRequeues(key) < maxRetries {
-		logrus.Warnf("Error syncing Longhorn backing image %v: %v", key, err)
+		logrus.WithError(err).Warnf("error syncing Longhorn backing image %v", key)
```

Comment:
Violate the existing convention. Please check https://github.com/longhorn/longhorn-manager/pull/1804#issuecomment-1480707966